monty/shutil.py

```diff
@@ -64,7 +64,9 @@ def gzip_dir(path: str | Path, compresslevel: int = 6) -> None:
 
 
 def compress_file(
-    filepath: str | Path, compression: Literal["gz", "bz2"] = "gz"
+    filepath: str | Path,
+    compression: Literal["gz", "bz2"] = "gz",
+    target_dir: str | Path = "",
```

Comment:
I would recommend making the default "." (for current working directory), as "" is not clear.

Same comment in the function below.

Reply:
If we change the default to ".", the default behaviour would change (from compress files in place to compress files to `cwd`)? Of maybe use `None` as default for in-place compression?

Reply:
Ah, sorry about that. You're right. The default should be `None`.

Reply:
All good. Would change that.